test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionProgressIT.java

```diff
@@ -97,14 +104,90 @@ public void testProgress() throws Exception {
       client.tableOperations().attachIterator(table1, setting,
           EnumSet.of(IteratorUtil.IteratorScope.majc));
       log.info("Compacting table");
-      compact(client, table1, 2, "DCQ1", true);
+      compact(client, table1, 2, QUEUE1, true);
       verify(client, table1, 2, ROWS);
 
       log.info("Done Compacting table");
       compactionFinished.set(true);
       checkerThread.join();
 
       verifyProgress();
+    } finally {
+      getCluster().getClusterControl().stopAllServers(ServerType.COMPACTOR);
+      getCluster().getClusterControl().stopAllServers(ServerType.COMPACTION_COORDINATOR);
+    }
+  }
+
+  @Test
+  public void testProgressWithBulkImport() throws Exception {
+    /*
+     * Tests the progress of an external compaction done on a table with bulk imported files.
+     * Progress should stay 0-100. There was previously a bug with the Compactor showing a >100%
+     * progress for compactions with bulk import files.
+     */
+    String[] tableNames = getUniqueNames(2);
+    String tableName1 = tableNames[0];
+    String tableName2 = tableNames[1];
+
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
```

Comment:
Does this test fail w/o the new changes?

Reply:
Yes